File: Python/FactoryPattern/DatabaseAdapter/db/mysql_database.py
from db.database import Database


class MySQLDatabase(Database):
    def __init__(self):
        print("======== MySQL ========")

    def create_table(self, table_name: str) -> None:
        print("Creating table: " + table_name)

    def query_table(self, table_name: str, fields: str) -> None:
        print("Querying Table: " + table_name + " for fields " + fields)

File: FactoryPattern/Java/DatabaseAdapter/DatabaseFactory/src/com/laxmena/db/MySQLDatabase.java
package com.laxmena.db;

public class MySQLDatabase implements Database {
    public MySQLDatabase() {
        System.out.println("============= MySQL Database =============");
    }
    @Override
    public void createTable(String tableName) {
        System.out.println("Creating MySQL table: " + tableName);
    }

    @Override
    public void queryTable(String tableName, String query) {
        System.out.println("Querying Table " + tableName + " for " + query);
    }
}
